test/functionalTest/cases/3004_v2_reg_create/v2_reg_create_v1_forward.test

```diff
@@ -26,17 +26,18 @@ Create registration using APIv2 and provoke v1 forwards
 --SHELL-INIT--
 dbInit CB
 dbInit CP1
-brokerStart CB
+brokerStart CB 100,186 IPV4
 brokerStart CP1
 
 --SHELL--
 
 #
 # 01. Register entity E1/A1 (using APIv2) for CP1
-# 02. Create E1/A1 in CP1
-# 03. GET E1/A1 from main broker (forward made to CP1)
-# 04. UPDATE E1/A1 in main broker (forward made to CP1)
-# 05. GET E1/A1 directly from CP1, make sure updates are there
+# 02. See the registration in mongo
```

Comment:
To be sure, this test change is only a new step in which mongo document is inspected, isn't it?

Reply:
That's it

Reply:
We don't use to include mongoCmd in .test except if we don't have other option. Ttypically, when we cannot check something using the API. But I understand this is not the case, as the API allows to check the registration with GET /v2/registration/{id}.

Having said that, I don't mind to keep it, but will be probably removed in the future if we have problems with it (the DB model can change while the API keeps the same and a good e2e test should be prepared for that).

Reply:
As what we stored in mongo was incorrect (for regs with idPattern) I needed this step to see what was really happening. Decided to keep it, didn't hurt. But yes, if it "starts hurting", no problem with removing it.

NTC